dataland-frontend/src/components/forms/CreateEUTaxonomyForNonFinancials.vue

```diff
@@ -1,139 +1,627 @@
+                              :explanation="euTaxonomyKpiInfoMappings[detailCashFlowType] ?? ''"
+                            />
+                            <KPIfieldSet
+                              :kpiInfoMappings="euTaxonomyKpiInfoMappings"
+                              :kpiNameMappings="euTaxonomyKpiNameMappings"
+                              :toggleDataAvailable="false"
+                            />
+                          </div>
+                        </FormKit>
+                      </div>
+                    </FormKit>
+                  </div>
+                </div>
+
+                <div class="uploadFormSection">
+                  <div class="col-3 p-3 topicLabel">
+                    <h4 id="opex" class="anchor title">OpEx</h4>
+                  </div>
+
+                  <!-- OpEx -->
+                  <div data-test="opexSection" class="col-9 formFields">
+                    <h3 class="mt-0">OpEx</h3>
+                    <FormKit name="opex" type="group">
+                      <div
+                        v-for="detailCashFlowType of euTaxonomyKPIsModel.euTaxonomyDetailsPerCashFlowType"
+                        :key="detailCashFlowType"
+                        :data-test="detailCashFlowType"
+                        class="form-field"
+                      >
+                        <FormKit :name="detailCashFlowType" type="group">
+                          <div class="form-field">
+                            <UploadFormHeader
+                              :name="euTaxonomyKpiNameMappings[detailCashFlowType] ?? ''"
+                              :explanation="euTaxonomyKpiInfoMappings[detailCashFlowType] ?? ''"
+                            />
+                            <KPIfieldSet
+                              :kpiInfoMappings="euTaxonomyKpiInfoMappings"
+                              :kpiNameMappings="euTaxonomyKpiNameMappings"
+                              :toggleDataAvailable="false"
+                            />
+                          </div>
+                        </FormKit>
+                      </div>
+                    </FormKit>
+                  </div>
+                </div>
+
+                <div class="uploadFormSection">
+                  <div class="col-3 p-3 topicLabel">
+                    <h4 id="revenue" class="anchor title">Revenue</h4>
+                  </div>
+
+                  <!-- Revenue -->
+                  <div data-test="revenueSection" class="col-9 formFields">
+                    <h3 class="mt-0">Revenue</h3>
+                    <FormKit name="revenue" type="group">
+                      <div
+                        v-for="detailCashFlowType of euTaxonomyKPIsModel.euTaxonomyDetailsPerCashFlowType"
+                        :key="detailCashFlowType"
+                        :data-test="detailCashFlowType"
+                        class="form-field"
+                      >
+                        <FormKit :name="detailCashFlowType" type="group">
```

Comment:
The input vields under "total amount" are for percentages, that feels like it should not be like this. Feels that they should be for total values.

Reply:
ok